packages/react/src/drawer/docs/overview.mdx

```diff
@@ -347,3 +347,70 @@ By default, Drawer’s overlay defaults to 80% opacity, however, by adding `mute
 	);
 };
 ```
+
+## Focus custom element on close
+
+When the drawer closes, focus is returned to the element that triggered its opening. You can override this behaviour by passing a custom element to the `elementToFocusOnClose` prop. This is useful when an action performed in the drawer causes an error and focus should be returned to an alert on close.
```

Comment:
Perhaps something like this?
```suggestion
By default, when a drawer closes, focus is returned to the element that opened it. Should you need to override this behaviour, pass a element ref to the `elementToFocusOnClose` prop. This can be used in situations such as when an action performed in the drawer causes an error on the main page, thus requiring the user’s focus to be sent to the alert instead.
```

Reply:
I tried to spice it up by not re-using the "By default" opener that already exists on this docs page.